src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -430,7 +434,7 @@ def simulated_test_runner():
             )
 
             # Only validate on the blue worlds
-            blue_full_system_proto_unix_io.register_observer(World, runner.world_buffer)
+            # blue_full_system_proto_unix_io.register_observer(World, runner.world_buffer)
```

Comment:
this is the line that causes issues. we register as a World observer twice (the first one in the tbots test runner) so we receive each world twice. in the test fixture, we run one world, run validation, advance time, then run another world and validate and advance time so because of duplicated worlds we only validate the first 7.5 seconds of worlds in a 15 second test